go.mod

```diff
@@ -1,17 +1,29 @@
 module github.com/portworx/kvdb
 
-go 1.17
+go 1.20
+
+require (
```

Comment:
why is this section apart from the require section below? 
If you run `go mod tidy` or `go mod vendor`, they should organize it for you.

Reply:
tried both but I think it doesn't organise automatically, however I think this wouldn't be a problem
Also, I have updated it by hand now